config/calibrate.py

```diff
@@ -42,28 +38,12 @@
         matchConfig.matcher.maxMatchDistArcSec = 2.0
         matchConfig.sourceSelector.active.excludePixelFlags = False
 
-# Set to match defaults currently used in HSC production runs (e.g. S15B+)
-config.catalogCalculation.plugins['base_ClassificationExtendedness'].fluxRatio = 0.95
-
-# Demand astrometry and photoCal succeed
-config.requireAstrometry = True
-config.requirePhotoCal = True
-
+# to assist with debugging
```

Comment:
Please make this into a complete sentence to indicate exactly what you intend will assist with debugging, starting with a capital letter and ending with a full stop.